WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2158,10 +2173,12 @@ static GSourceFuncs _handlerIntervention =
                 Core::SystemInfo::SetEnvironment(_T("MSE_MAX_BUFFER_SIZE"), _config.MSEBuffers.Value(), !environmentOverride);
             }
 
+#ifndef WEBKIT_MEMORY_PRESSURE_API
             // Memory Pressure
             if (_config.MemoryPressure.Value().empty() == false) {
```

Comment:
Hi @modeveci .  I think it'd be cleaner if memory pressure value is just reinterpreted here in 2.38 case. I mean instead of setting environment variable, you could parse it and set the settings using new API in the worker later. 

Changing browser config will require changes in build recipes in different meta layers, and can leave browser in wrong configuration if those aren't updated....

Reply:
Thank you for your feedback @emutavchi. @MFransen69 could you please review this request?